Cargo.toml

```
@@ -52,6 +52,6 @@ chrono = { version = "0.4", default-features = false }
 assert-json-diff = "2.0.2"
 mockall = "0.11.4"
 serial_test = "2.0.0"
-k8s-openapi = { version = "0.18.0", default-features = false, features = [ "v1_26" ] }
+k8s-openapi = { version = "0.19.0", default-features = false, features = [ "v1_26" ] }
```

```suggestion
k8s-openapi = { version = "0.19.0", default-features = false, features = [ "latest" ] }
```

Use the new `latest` feature on k8s-openapi 0.19.0. This will track the latest supported k8s version. At the time of this commit, 1.27.